Makefile

```diff
@@ -295,6 +295,9 @@ $(GOPATH)/bin/mockery:
 	mockery -version
 
 $(GOPATH)/bin/controller-gen:
+	# The vendored versions needed by controller-gen are different than the ones that
+	# are installed previously installed by the makefile. Remove the previous one to ensure the new one is fetched
+	rm -rf vendor
```

Comment:
Not quite sure what's wrong here yet, but this is a workaround to fix `make codegen`.

Note: this only started happening after I upgraded go to `v1.15`